packages/react-components/react-tree-preview/stories/Tree/TreeManipulation.stories.tsx

```diff
@@ -26,23 +26,21 @@ const defaultSubTrees: ItemProps[][] = [
   ],
 ];
 
-const CustomTreeItem = ({
-  item,
-  isItemRemovable,
-  onRemoveItem,
-}: {
-  item: HeadlessTreeItem<ItemProps>;
-  onRemoveItem: (id: string) => void;
-  isItemRemovable: boolean;
-}) => {
-  const { content, ...treeItemProps } = item.getTreeItemProps();
+type CustomTreeItemProps = TreeItemProps & {
+  onRemoveItem?: (value: string) => void;
+};
+
+const CustomTreeItem = ({ onRemoveItem, ...props }: CustomTreeItemProps) => {
+  const level = props['aria-level'] as number;
```

Comment:
```suggestion
  const level = Number(props['aria-level']);
```

let's not use cast if we can avoid it